tinydancer/src/main/java/com/codemonkeylabs/fpslibrary/ui/TinyCoach.java

```diff
@@ -54,10 +55,17 @@ public TinyCoach(Application context, FPSConfig config) {
 
     private void addViewToWindow(View view) {
 
+        int permissionFlag;
+        if(android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Maybe encapsulate in PermissionCompat.gerFlag()

Reply:
will do!